DEV.md

```diff
@@ -43,21 +43,21 @@ E.g:
 This command runs the unit tests from the `build-module` module.
 
 ```bash
-./mill 'build-module.test'
+./mill 'build-module[].test'
 ```
 
 If you want to run unit tests for another module, set `module_name` to the name of the module from which you want to run
 the unit tests:
 
 ```bash
-./mill 'module_name.test'
+./mill 'module_name[].test'
```

Comment:
not all modules are cross-compiled.

Reply:
Yes.
https://mill-build.org/mill/fundamentals/cross-builds.html#_cross_modules_defaults

Reply:
Done